Dockerfile

```diff
@@ -25,6 +25,10 @@ ENV PHP_ERROR_LOG /var/log/lighttpd/error.log
 COPY ./start.sh /
 COPY ./bash_functions.sh /
 
+# Add PADD to the container, too.
+ADD https://raw.githubusercontent.com/pi-hole/PADD/master/padd.sh /padd.sh
```

Comment:
I think we could use `https://install.padd.sh` here, since we already  have a [PR to change the URL](https://github.com/pi-hole/PADD/pull/213)

Reply:
```suggestion
ADD https://install.padd.sh /padd.sh
```